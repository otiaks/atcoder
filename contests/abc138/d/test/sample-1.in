4 3
1 4
1 3
2 3
2 10
1 100
3 1