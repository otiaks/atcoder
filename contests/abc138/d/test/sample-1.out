100 111 101 100
